src/extensions/install/install.cmd.tsx

```diff
@@ -12,15 +18,29 @@ export default class InstallCmd implements Command {
   shortDescription = '';
   options = [];
 
-  constructor(private workspace: Workspace) {}
+  constructor(private workspace: Workspace, private packageManager: PackageManager) {}
 
   // TODO: remove this ts-ignore
   // @ts-ignore
   async render() {
     try {
       const components = await this.workspace.list();
       const isolatedEnvs = await this.workspace.load(components.map(c => c.id.toString()));
-      return <Color green>Successfully installed {isolatedEnvs.length} components</Color>;
+      const packageManagerName = this.workspace.consumer.config.packageManager;
+
+      const packageJson = await fs.readJson(path.join(process.cwd(), 'package.json'));
+      packageJson.dependencies = packageJson.dependencies || {};
```

Comment:
what about dev / peer? are they not here by purpose?

Reply:
Yeah, we just need this one here because we're adding to it and want to make sure it exists (it's optional in the package.json template).